homeassistant/components/sensor/device_condition.py

```diff
@@ -81,6 +105,7 @@
             vol.Required(CONF_ENTITY_ID): cv.entity_id,
             vol.Required(CONF_TYPE): vol.In(
                 [
+                    CONF_IS_AQI,
```

Comment:
I think that the entity needs to have a unit of measurement to work with device conditions. How do we solve this for AQI?

Reply:
Is the best thing right now just to not have device conditions for AQI? It looks like it's not mandatory for a device class as MONETARY doesn't have it.

Alternatively I could amend `async_get_condition_capabilities` to allow `unit_of_measurement` to be `None`. In that case, suffix could just be an empty string? If that's the route we want to go down i'd probably pull the AQI device condition out of this PR anyway and into a second PR that implemented the device conditions change.

(Part of me briefly considered if AQI *is* the unit, but I don't think so. You can't /really/ have 1.5 AQI. What would that mean? As it's really just assigning a number to a range and saying that number means "Excellent". I did wonder whether there was such a think as a "text sensor" so that I could instead just have a state of "Good", "Moderate", "Unhealhty" etc.  But I do think knowing the average/min/max AQI for your room is useful).

Reply:
Not supporting device condition for AQI is ok for now I think.

Reply:
Removed in latest commit 👍